docs/index.md

```diff
@@ -51,20 +51,21 @@ Usage:
   dashboard-linter lint [dashboard.json] [flags]
 
 Flags:
-  -c, --config string   path to a configuration file
-      --fix             automatically fix problems if possible
-  -h, --help            help for lint
-      --strict          fail upon linting error or warning
-      --verbose         show more information about linting
+  -c, --config string         path to a configuration file
+      --fix                   automatically fix problems if possible
+  -h, --help                  help for lint
+  -m, --matcher stringArray   matcher required to be present in all selectors, e.g. 'instance=~"$instance"' or 'cluster="$cluster"', can be specified multiple times (default ["instance=~""$instance""","job=~""$job"""])
+      --stdin                 read from stdin
+      --strict                fail upon linting error or warning
+      --verbose               show more information about linting
 ```
 
 # Rules
 
 The linter implements the following rules:
 
 * [template-datasource-rule](./rules/template-datasource-rule.md) - Checks that the dashboard has a templated datasource.
-* [template-job-rule](./rules/template-job-rule.md) - Checks that the dashboard has a templated job.
-* [template-instance-rule](./rules/template-instance-rule.md) - Checks that the dashboard has a templated instance.
```

Comment:
In the interest of providing a smooth transition for existing users, let's not delete these yet. Those who wish to use the new rule can add exclusions to their `.lint` file.